cypress/support/pages/course/pageCourses.js

```diff
@@ -103,6 +103,83 @@ class Courses {
 		'[data-testid="videoconference-config-dialog-title"]';
 	static #bbbDialogBoxCancelButtonNBC = '[data-testid="dialog-cancel"]';
 	static #bbbDisabledCheckBoxNBC = '[data-testid="videoconf_checkbox"]';
+	static #oldToolsTabInCourseDetail = '[data-testid="old-tools-tab"]';
+	static #listToolsCourse= '[data-testid="course_tool_list"]'
+	static #modalContent = '[data-testid="modal_content"]'
+  static #addBBBButton = '[data-testid="submit-btn-add-bbb-tool-modal"]'
+  static #modalContentCreateVideoconf = '[class="modal fade create-videoconference-modal in"]'
```

Comment:
1- done 
2- I used this class because data-testid - is not constant - it depends on the course, part data-testid  is the course title
![image](https://github.com/hpi-schul-cloud/e2e-system-tests/assets/118526182/ad036da9-7582-46ee-a904-d3464d1e4c77)

Reply:
We have changed date-testid from variable to constant.  `'[data-testid^="btn-cancel-"]` >>>>`data-testid="btn-cancel"` 
It is now much easier to do. This change has been checked by the developers and does not create a conflict.